wiki/Accuracy/en.md

```diff
@@ -1,132 +1,38 @@
-|:------------------------:|:-------------------------------------------------------------------------------------------:|
-| **Total points of hits** | `((Number of Miss * 0) + (Number of 100(GOOD) * 0.5) + (Number of 300(GREAT) * 1)) * 300` |
-| **Total number of hits** | `Number of Miss + Number of 100 + Number of 300     `                                         |
-
-### osu!catch
-
-In [osu!catch][osu!catch wikilink], the song accuracy is calculated as the number of non-spinner objects collected divided by the total number of non-spinner objects; fruits, large drops and small drops (droplets) all have the same value.
-
-In other words: `Accuracy = Total numbers of fruits caught / Total number of fruits`
-
-| Term                               | Formula                                                                                              |
-|:----------------------------------:|:----------------------------------------------------------------------------------------------------:|
-| **Total numbers of fruits caught** | `Number of droplets + Number of drops + Number of fruits`                                              |
-| **Total numbers of fruits**        | `Number of misses + Number of miss droplets + Number of droplets + Number of drops + Number of fruits` |
-
-"Bananas" (spinner's fruits) do not count.
-
-Note that if using the API to calculate the accuracy, number of droplets is under **count50** and number of miss droplets is under **countkatu**.
-
-### osu!mania
-
-In [osu!mania][osu!mania wikilink], accuracy is calculated similarly to [osu!standard][osu! wikilink].
-
-In other words: `Accuracy = Total points of hits / (Total number of hits * 300)`
-
-| Term                     | Formula                                                                                                                       |
-|:------------------------:|:-----------------------------------------------------------------------------------------------------------------------------:|
-| **Total points of hits** | `(Number of 50s * 50 + Number of 100s * 100 + Number of 200s * 200 + Number of 300s * 300 + Number of rainbow 300s * 300)`    |
-| **Total number of hits** | `(Number of misses + Number of 50s + Number of 100s + Number of 200s + Number of 300s + Number of rainbow 300s)`              |
-
-
-Note that MAX (or rainbow 300) and 300 are both worth the maximum for calculating accuracy, despite a MAX being worth more in terms of score than a 300.
-
-## Results screen
+Accuracy is the measurement used to determine a player's skill. Even so, there are three types of accuracy that a player can have. One of them being the beatmap's accuracy which is dependent on hit scores gained. Another being the player's overall accuracy that is weighed to allow better scores to stand out more. And lastly, the player's pp's accuracy which is dependent on the submitted score's accuracy.
```

Comment:
Accuracy is the measurement used to determine a player's skill
not skill, more like consistency

Reply:
`Accuracy is the measurement used to determine a player's consistency.` :question: